Festivity/Program.cs

```diff
@@ -22,8 +22,9 @@ public static void Main(string[] args)
             Console.SetWindowSize(150, 35);
             void homepage()
             {
-                int option = 0;
-                string[] consoleOptions = new string[] { "Register", "Login", "Festivals", "Register festival", "Exit" };
+                int Option = 0;
+                string[] ConsoleOptions = new string[] { "Register", "Login", "Festivals", "Register festival", "Exit", "Festival Page" };
+
```

Comment:
In de rest van de code wordt geloof ik option en consoleOptions gebruikt sinds vorige week